packages/blockchain/README.md

```diff
@@ -180,6 +180,10 @@ The blockchain library now allows for blob transactions to be validated and incl
 
 **Note:** Working with blob transactions needs a manual KZG library installation and global initialization, see [KZG Setup](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/tx/README.md#kzg-setup) for instructions.
 
+### EIP-6110/EIP-7002 Requests Support
```

Comment:
Also 7251 here?

Reply:
Have substantially overhauled this to be generally more future-proof (in the style of our generic supported-EIPs lists) 🙂 

![grafik](https://github.com/user-attachments/assets/c0b2eecb-813f-4f62-928d-f762d4908414)

Reply:
(also for VM)